antora-playbook.yml

```diff
@@ -67,6 +67,8 @@ asciidoc:
   sourcemap: false
   attributes:
     icons: font
+  extensions:
+    - '@asciidoctor/tabs'
```

Comment:
TODO: update `ui.bundle.url` when new release of bonita-documentation-theme is ready